osu.Game.Rulesets.Taiko/Difficulty/Preprocessing/TaikoDifficultyHitObject.cs

```diff
@@ -17,21 +17,22 @@ namespace osu.Game.Rulesets.Taiko.Difficulty.Preprocessing
     /// </summary>
     public class TaikoDifficultyHitObject : DifficultyHitObject
     {
+        private readonly IReadOnlyList<TaikoDifficultyHitObject>? monoDifficultyHitObjects;
+        public readonly int MonoIndex;
+        private readonly IReadOnlyList<TaikoDifficultyHitObject> noteObjects;
+        public readonly int NoteIndex;
```

Comment:
The list fields here could really do with some xmldocs describing these, I find myself needing to check what `monoDifficultyHitObjects` and `noteObjects` are.

Additional nit: shouldn't it be `monoObjects` or `noteDifficultyHitObjects` for consistency with eachother?

Reply:
Applied in 4433f902ea4a20a6ec7bbcf70ee03ce1c26134a3